Parsers/ASimDns/Parsers/vimDnsMicrosoftNXlog.yaml

```diff
@@ -190,8 +192,62 @@ ParserQuery: |
     , 29,'LOC'
     , 30,'NXT'
     , 31,'EID'
-    , 32,'NIMLOC'
+    , 32,'NB'
     , 33,'SRV'
+    , 34,'ATMA'
```

Comment:
@jszigetvari-nxlog I noticed that some of the types are from obsolete list mentioned here: https://en.wikipedia.org/wiki/List_of_DNS_record_types
I think we should avoid adding the obsolete ones.

Reply:
fixed

Reply:
can you please revalidate I could still see 34, 38 in table which is part of Obselete list in Wiki

Reply:
done.

Reply:
changes not visible.

Reply:
this is from the last commit:
![screenshot](https://github.com/Azure/Azure-Sentinel/assets/117363357/6e583bc1-4890-4687-a8c5-4431da05b593)

Reply:
@vakohl I undid most of the removals, because the entries removed at that time were still present in the list you recommended, and thus this is the new list.